From edde38979dd125370a0f69d91d6e0f3f7430e791 Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Sat, 22 Nov 2025 21:48:54 +0100
Subject: [PATCH] CMakeLists: update min version to 3.10 for CMake >= 4.0

Update minimum version of CMake to 3.10 to support CMake >= 4.0 version.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.10)
 
 project(bluld CXX)
 
